### ISSUE TITLE
Formatting as a code Action?

### ISSUE THREAD

**Issue by vaeng** [exercism/sqlite#13]
Do you think it would be useful to have automatic code formatting for out `.SQL` files?
There is https://github.com/andialbrecht/sqlparse, which could be used to build a represented as well.

**Comment by IsaacG** [comment]
Absolutely! If there is a way to defer formatting choices to a tool which makes things consistent, I think that would be great.

**Comment by vaeng** [comment]
I added an initial python script to format our SQL files: #16
